### PR TITLE
[TASK] Make the AbstractPlugin @internal

### PR DESCRIPTION
**Pull request by linawolf**
resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/171
releases: main

### CODE REVIEW

File: Documentation/ExtensionArchitecture/HowTo/FrontendPlugin/AbstractPlugin.rst [hunk 1]
```diff
@@ -1,170 +1,96 @@
...
-   :php:`string`: The extension key in snake_case, for example :php`'my_extension'`
-
-
+:orphan:
+
+=========================
+AbstractPlugin (internal)
+=========================
+
+..  versionchanged:: 12.ß
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/FrontendPlugin/AbstractPlugin.rst:7]:
```suggestion
..  versionchanged:: 12.0
```

File: Documentation/ExtensionArchitecture/HowTo/FrontendPlugin/Index.rst [hunk 2]
```diff
@@ -18,13 +18,6 @@ For pure output it is often sufficient to use a
 For scenarios with user input and or complicated data operations consider
 using :ref:`Extbase <extbase>`.
 
-Legacy frontend plugins without Extbase, so called "pi-based plugins" are based
-on the :ref:`AbstractPlugin <abstractplugin>`. It is not recommended anymore
-to use the AbstractPlugin as base for new frontend plugins. The Core does not
-use it anymore and only few third party extensions still use it.
+It is also possible to create a front end plugin using Core functionality only.
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/FrontendPlugin/Index.rst:21]:
```suggestion
It is also possible to create a frontend plugin using Core functionality only.
```

File: Documentation/ExtensionArchitecture/HowTo/FrontendPlugin/Index.rst [hunk 3]
```diff
@@ -18,13 +18,6 @@ For pure output it is often sufficient to use a
...
 
-.. toctree::
-   :titlesonly:
-   :glob:
-
-   *
+..  todo: Document how to create a frontend plugin without Extbase.
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/HowTo/FrontendPlugin/Index.rst:23]: I assume, this will be a follow-up PR?

**Review by brotkrueml — APPROVED** [review]

File: Documentation/ExtensionArchitecture/HowTo/FrontendPlugin/Index.rst (hunk 3 above)
Comment (linawolf) [reply Documentation/ExtensionArchitecture/HowTo/FrontendPlugin/Index.rst:23]: yes

**Pull request merged**